crates/turborepo-auth/src/lib.rs

```diff
@@ -89,6 +89,30 @@ pub async fn read_or_create_auth_file(
     Ok(auth_file)
 }
 
+/// Sync version of `read_or_create_auth_file`.
```

Comment:
I think the comment needs to be updated as there are behavior differences between this and `read_or_create_auth_file` (and also they're both now sync).